docs/dev/fundamentals/reading-games-code.md

```diff
@@ -0,0 +1,97 @@
+---
+description: This article teaches how to use ILSpy in order to read game's code
+---
+
+# Reading game's code
+## Introduction
+Chances are, early on in your modding journey you will be met by quite a conundrum - You want to know how something in the game works, but there is an issue! You cannot just open the game's files to find it *(you silly goose)*. 
+
+You see, the way Unity games work, is that their code gets compiled into a .dll file, which you cannot normally open. 
+
+Fortunately, there is something called a "decompiler". What it does, is it grabs game's code and reverses the process, resulting in readable code. While it is not perfect, it gets it's job done.
+
+::: warning
+Please keep in mind that while decompiling and reading the code is *not* illegal, distributing said decompiled code *is* illegal. Same applies for sharing the game's files. 
+
+What this means is that you **CANNOT** bundle any of the game's files with your mods.
+:::
+
+## ILSpy
```

Comment:
Consider renaming section to "Decompilers" and list other decompilers as well (namely dnSpyEx and dotPeek). You can then state that this tutorial uses ILSpy but all three practically are used in the same way.